eng/pipelines/performance/templates/build-perf-sample-apps.yml

```diff
@@ -1,35 +1,21 @@
 parameters:
   osGroup: ''
-  osSubgroup: ''
-  archType: ''
-  buildConfig: ''
-  runtimeFlavor: ''
-  helixQueues: ''
-  targetRid: ''
+  flavor: 'mono' # Currently only used for Android Hello World app
   nameSuffix: ''
-  platform: ''
-  shouldContinueOnError: ''
-  rootFolder: ''
-  includeRootFolder: ''
-  displayName: ''
-  artifactName: ''
-  archiveExtension: ''
-  archiveType: ''
-  tarCompression: ''
   hybridGlobalization: False
 
 steps:
 # Build Android sample app
   - ${{ if eq(parameters.osGroup, 'android') }}:
```

Comment:
We should add preserving .binlog as an artifact of Android sample, like we do for iOS samples